### PR TITLE
✨ 880 - Consulting pages - Video Production template

### LINKED ISSUE #880: 📄 Consulting - Video Production Services
We need a new template for these pages

Investigate if we can put these pages on

`consulting/video-production/*`

if not, we can put them on

`/video-production`

URL: https://www.ssw.com.au/ssw/Consulting/Video-Production/Video-Production-Product-Services.aspx

```[tasklist]
### Tasks
- [x] migrate this page.
- [x] Update links on https://www.ssw.com.au/consulting/video-production
- [x] Add redirect to Front Door
- [x] Check if there is a megamenu link to update
```

### PR DESCRIPTION
**Pull request by jackreimers**
Migrates https://www.ssw.com.au/ssw/Consulting/Video-Production/Video-Production-Product-Services.aspx to https://www.ssw.com.au/consulting/video-production/product-videos

 #880 

Affected routes:
- /consulting/video-production/product-videos

Add done video, screenshots
image
image
image

### CODE REVIEW

File: content/video-production/homepage-videos.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+seo:
+  title: Homepage Videos
+  description: ''
```
Comment (bradystroud) [inline content/video-production/homepage-videos.mdx]: bad SEO - copy meta desc from old page or make one up or ask @camillars

File: pages/consulting/video-production/[filename].tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    query: props.query,
+    variables: props.variables,
+  });
+
+  const bookingButtonProps = {
+    buttonText: data.global.bookingButtonText,
+  };
+
+  return (
+    <Layout>
```
Comment (bradystroud) [inline pages/consulting/video-production/[filename].tsx]: Need to add the SEO stuff

copy consulting template

https://github.com/SSWConsulting/SSW.Website/blob/656c9ac9c78b7fb250a7c33bf61f85eebe521f68/pages/consulting/%5Bfilename%5D.tsx#L59-L60

```suggestion
  return (
        <SEO seo={props.seo} />
    <Layout>
```

File: content/video-production/homepage-videos.mdx
Comment (bradystroud) [inline content/video-production/homepage-videos.mdx:1]: rename from `homepage-videos.mdx` to `product-videos.mdx`

https://www.ssw.com.au/rules/change-from-x-to-y/

File: pages/consulting/video-production/[filename].tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+          path={removeExtension(props.variables.relativePath)}
+          suffix={data.global.breadcrumbSuffix}
+          title={data.videoProduction.seo?.title}
+          seoSchema={data.videoProduction.seo}
+        />
+      </Section>
+
+      <Section className="w-full" color="black">
+        <Booking {...data.videoProduction.booking}>
+          <BookingButton {...bookingButtonProps} />
```
Comment (bradystroud) [inline pages/consulting/video-production/[filename].tsx]: Fix this - error on click

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]
🛶 lgtm

File: content/video-production/homepage-videos.mdx (hunk 1 above)
Comment (camillars) [reply content/video-production/homepage-videos.mdx]: Hi @jackreimers 

the SEO implemented does not describe this page. :(

1. please use the below
title - Video Production for Business & Product Success
description - Make your brand stand out with captivating product videos that impress your target audience and leave a lasting impact. (119 characters)

**Pull request merged**